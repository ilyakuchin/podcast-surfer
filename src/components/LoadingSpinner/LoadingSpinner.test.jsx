import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import LoadingSpinner from './LoadingSpinner';

configure({ adapter: new Adapter() });

describe('Test LoadingSpinner Component', () => {
  test('should render correctly', () => {
    expect(toJson(shallow(<LoadingSpinner />))).toMatchSnapshot();
  });
});
