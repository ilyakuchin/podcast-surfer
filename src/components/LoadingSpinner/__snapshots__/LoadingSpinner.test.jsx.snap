// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test LoadingSpinner Component should render correctly 1`] = `
<div
  className="loader"
/>
`;
